CHANGELOG.md

```diff
@@ -18,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fix publish script to use npm version
 - Add stage to saucelabs session name for integration tests
+- Optimized call requests to the cloudwatch
```

Comment:
Isn't this a new thing you are adding to the demo app. So you are not fixing anything here right.